keyboards/an_achronism/tetromino/config.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 an_achronism (@an-achronism)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define WS2812_PIO_USE_PIO1 // Use PIO1 on Pi Pico for RGB LEDs
```

Comment:
```suggestion

```
You shouldn't need to define this.

Reply:
I thought I should based on nothing much other than this, which seemed to be telling me that I should be defining it, but didn't appear to explain why: [https://docs.qmk.fm/#/ws2812_driver?id=pio](https://docs.qmk.fm/#/ws2812_driver?id=pio). But I can't say I have a strong justification for either including it or not; I'm not sure I follow precisely what the intention of it is, based on the documentation as it currently exists. Are you able to clarify at all, maybe? I'm guessing the option exists purely in case the PIO0 peripheral is busy doing something else for whatever reason, which it might not be here.

Reply:
Without testing for any significant duration, it certainly appears to work fine without that define so I guess I'll remove it! This might just be a case of tweaking docs for a bit of context: I read it like "you need to define the following for this to work optimally" but I am wondering if perhaps the intention was instead "you can define this if you want". Will remove now, thanks!

Reply:
Yeah you're not the first person to think it needs defining, so maybe a documentation edit is in order. I only point it out as it's likely to get copy and pasted.

Reply:
I might look into trying to tidy up some documentation stuff at some point, there were a few bits and bobs that were kind of confusing for a newbie like m'self but I didn't necessarily understand what I was reading enough to know what info I was missing or possibly misunderstanding. Having built a slightly better understanding now (largely thanks to your help, which I'm very grateful for) I might be in a somewhat better position to help with it now.